bbb-common-message/project/Dependencies.scala

```diff
@@ -7,21 +7,21 @@ object Dependencies {
 
   object Versions {
     // Scala
-    val scala = "2.12.8"
+    val scala = "2.12.15"
     val junit = "4.12"
     val junitInterface = "0.11"
     val scalactic = "3.0.3"
 
     // Libraries
-    val akkaVersion = "2.5.19"
-    val gson = "2.8.5"
-    val jackson = "2.9.7"
-    val sl4j = "1.7.25"
-    val red5 = "1.0.10-M9"
-    val pool = "2.8.0"
+    val akkaVersion = "2.6.17"
+    val gson = "2.8.9"
+    val jackson = "2.13.0"
+    val sl4j = "1.7.32"
+    val red5 = "1.2.10"

```

Comment:
I believe this is obsolete too - we don't use red5 anymore

Reply:
True red5 can be safely removed.

Reply:
Taken care of!